node/Makefile

```diff
@@ -5,15 +5,21 @@ LOTUS_PATH:=extern/lotus
 GOCC?=go
 
 submodule:
-	git submodule set-branch -b feat/nv18-fevm node/extern/lotus
+	git submodule set-branch -b release/v1.20.0 node/extern/lotus
 	git submodule sync
 	git submodule update --init --recursive --remote
+
+gomod:
+	@echo "module github.com/filecoin-project/fevm-contract-tests/node" > go.mod
+	@echo "go 1.18" >> go.mod
+	@echo "replace github.com/filecoin-project/lotus => ./extern/lotus" >> go.mod
+	@echo "replace github.com/filecoin-project/filecoin-ffi => ./extern/lotus/extern/filecoin-ffi" >> go.mod
```

Comment:
Can use `go mod edit` as it wil perform validations for you, but this is fine too.

Reply:
ok, I'll leave it as is and change it later if necessary.